mysql_scripts/table_creation/FullDB.sql

```diff
@@ -33,12 +38,12 @@ CREATE TABLE LitmusQuestion (
 	litmusQuestionId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
 	text VARCHAR(255) NOT NULL,
 	orderIndex INTEGER NOT NULL,
-	tag VARCHAR(255) NOT NULL,
+	lookupName VARCHAR(255) NOT NULL,
```

Comment:
tag is wildly ambiguous in my opinion, switching to lookupName because it clearly describes that this field is used for lookups in this table based on a string value.